projects/common/src/time/page-time-range-preference.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from '@angular/core';
+import { isNil } from 'lodash-es';
+import { Observable } from 'rxjs';
+import { map, shareReplay, take } from 'rxjs/operators';
+import { NavigationService } from '../navigation/navigation.service';
+import { PreferenceService, StorageType } from '../preference/preference.service';
+import { TimeRange } from './time-range';
+import { TimeRangeService } from './time-range.service';
+
+@Injectable({ providedIn: 'root' })
+export class PageTimeRangePreferenceService {
+  private static readonly STORAGE_TYPE: StorageType = StorageType.Session;
```

Comment:
@Christian862  As per discussion with @aaron-steinfeld, I think we can change the storage type to "Local"